components/conversation/apiHelpers.js

```diff
@@ -49,7 +50,7 @@ const sendMessage = async (message, chatId) => {
   formData.append('senderId', message.user._id);
   formData.append('body', message.text);
   formData.append('date', message.createdAt);
-  await fetch('http://3.133.100.147:2550/add-message', {
+  return await fetch('http://3.133.100.147:2550/add-message', {
```

Comment:
Not sure if this is what happened here, but SonarCloud alerted me to the fact I was not returning my promises once and found that super helpful